CHANGELOG.md

```diff
@@ -1,3 +1,117 @@
+* **ComputeSystemSensitivity:** a task to estimate the system sensitivity ([ac3e5ab](https://github.com/radiocosmology/draco/commit/ac3e5ab8dd44cb8d9b22145fd179222ea9c52283))
+* **containerBase:** support for sel_* params ([#77](https://github.com/radiocosmology/draco/issues/77)) ([01c1676](https://github.com/radiocosmology/draco/commit/01c16763a5221d814dec567e70ef02b196873bbf))
+* **ContainerBase:** copy an existing container and share its data ([518db11](https://github.com/radiocosmology/draco/commit/518db119d585a466289704057799611171d4575f))
+* **containers:** Add input axis to weight dataset of StaticGainData. ([e6ca2d9](https://github.com/radiocosmology/draco/commit/e6ca2d9423de004c1cc5acb6fd4e80617d3b3ba2))
+* **containers:** create SystemSensitivity container ([cc8672c](https://github.com/radiocosmology/draco/commit/cc8672c0c174e411014921b575656c329b367ae5))
+* **core.containers:** add chunking and compression to containers ([12ece28](https://github.com/radiocosmology/draco/commit/12ece28ed2b1f542bbcfc33e899fe76415f7260a))
+* **core.containers:** Add compression to weights datasets and TimeStream. ([9ec6fbc](https://github.com/radiocosmology/draco/commit/9ec6fbc604575082e518e35d87b541b4a362350a))
+* **core.io:** add Truncate task. ([5a7f9f8](https://github.com/radiocosmology/draco/commit/5a7f9f8fb7570567e9cbb8100f9b666a79e0910a))
+* **delay:** Add baseline dependent delay cut ([9e797c3](https://github.com/radiocosmology/draco/commit/9e797c3fe98917ecfba4401db34f2b7a868d5a0d))
+* **delay:** Add telescope orientation option. ([b52b83b](https://github.com/radiocosmology/draco/commit/b52b83b1699897d41c02ee390101116e9954841a))
+* **delay:** Minor documentation clean-up ([34d257f](https://github.com/radiocosmology/draco/commit/34d257fc1a57c9f0cb898975af65a14fd456c083))
+* **ExpandProducts:** generate stack maps to match andata containers ([a7046ca](https://github.com/radiocosmology/draco/commit/a7046ca31a3d76a6aaa48ae8129f65b03d19c0b6))
+* **flagging:** add task that sets weights below threshold equal to zero ([d29a9e8](https://github.com/radiocosmology/draco/commit/d29a9e8dde236d751dd177f6b433a2598b86aba8))
+* **flagging:** quick RFI flagging for scattered TV ([c9fa971](https://github.com/radiocosmology/draco/commit/c9fa971655ed97fb75e4e3ea55d0b0db91971caf))
+* **gain.py:** add tasks SiderealGains and GainStacker ([9afa726](https://github.com/radiocosmology/draco/commit/9afa726f99797984f4d1d6919b8f8eadf0623d82)), closes [#20](https://github.com/radiocosmology/draco/issues/20)
+* **gaussiannoise:** add a first-pass add a gaussian noise dataset generator ([b974338](https://github.com/radiocosmology/draco/commit/b97433886150818cdf53ea32bdb59b4ddb2fff0d)), closes [#52](https://github.com/radiocosmology/draco/issues/52)
+* **gaussiannoise:** add local seed states for RandomGen ([4bec93a](https://github.com/radiocosmology/draco/commit/4bec93a1009945bb7e731905229e1add5937d25f)), closes [#52](https://github.com/radiocosmology/draco/issues/52)
```

Comment:
I think the changelog is incorrect. For instance I'm pretty sure this change was already in 20.2.0.

Reply:
Good catch. I forgot fetching the tags.